client/flutter/lib/generated/intl/messages_all.dart

```diff
@@ -15,43 +15,43 @@ import 'package:intl/intl.dart';
 import 'package:intl/message_lookup_by_library.dart';
 import 'package:intl/src/intl_helpers.dart';
 
-import 'messages_ru_RU.dart' as messages_ru_ru;
-import 'messages_es_ES.dart' as messages_es_es;
-import 'messages_en_GB.dart' as messages_en_gb;
 import 'messages_ar.dart' as messages_ar;
-import 'messages_fr_FR.dart' as messages_fr_fr;
 import 'messages_en.dart' as messages_en;
+import 'messages_en_GB.dart' as messages_en_gb;
```

Comment:
Unless this file has to be changed, please revert.

Reply:
This is done by the Flutter Intl plugin automatically.